config/config.example.yml

```diff
@@ -1,13 +1,757 @@
-channel_threads: 1
-feed_threads: 1
+#########################################
+#
+#  Database configuration
+#
+#########################################
+
+##
+## Database configuration with separate parameters.
+## This setting is MANDATORY, unless 'database_url' is used.
+##
 db:
   user: kemal
   password: kemal
   host: localhost
   port: 5432
   dbname: invidious
-# alternatively, the database URL can be provided directly - if both are set then the latter takes precedence
-# database_url: postgres://kemal:kemal@localhost:5432/invidious
-full_refresh: false
-https_only: false
+
+##
+## Database configuration using a single URI. This is an
+## alternative to the 'db' parameter above. If both forms
+## are used, then only database_url is used.
+## This setting is MANDATORY if 'db' is not set.
```

Comment:
"This setting is MANDATORY if 'db' is commented out"*

Reply:
Wouldn't it be better with a form like this (to be coherent between the two)? 
https://github.com/SamantazFox/invidious/blob/patch-1/config/config.example.yml#L9

Reply:
Yes, this is even better.